src/functions/ruler_skill_check.txt

```diff
@@ -0,0 +1,44 @@
+new effect ruler_skill_check = {
+	args = {
+		string type
```

Comment:
unfortunately i can't understand this part yet so i can't say anything about it kurashi check this file again to be sure

Reply:
I wrote it.